transcendence-app/src/auth/fortytwo/fortytwo-config.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface FortyTwoAuthConfig {
```

Comment:
I agree with @vdedios, and there is a auth module which just import 42 module. May only 42oAuth module is enough

Reply:
You cannot start symbol names with digits in many programming languages because of the ambiguity on wether symbols are symbols or literal numbers. I would prefer to follow it here, even if not strictly necessary (is it not necessary?).
TLDR, why have we been naming everything as ft_ rather than 42_ so far otherwise?

Reply:
`oauth42` :smile:

Reply:
OAuth42 sound like a good name. Done :+1: